src/main/java/mate/academy/Main.java

```diff
@@ -1,7 +1,47 @@
 package mate.academy;
 
+import java.math.BigDecimal;
+import java.util.List;
+import java.util.Objects;
+import mate.academy.dao.BookDao;
+import mate.academy.lib.Injector;
+import mate.academy.model.Book;
+
 public class Main {
+    private static final Injector INJECTOR = Injector.getInstance("mate.academy");
+    private static List<Book> books = List.of(
+            new Book("Core Java Volume I – Fundamentals", BigDecimal.valueOf(29.99)),
+            new Book("Effective Java", BigDecimal.valueOf(19.99)),
+            new Book("Java - The Complete Reference", BigDecimal.valueOf(35.45)));
+
     public static void main(String[] args) {
+        BookDao bookDao = (BookDao) INJECTOR.getInstance(BookDao.class);
+        //create
+        for (Book book : books) {
+            System.out.println("Created record in DB: "
+                    + bookDao.create(book));
+        }
+
+        //read
+        books = bookDao.findAll();
+        System.out.println(System.lineSeparator() + "Records in DB:");
+        books.forEach(System.out::println);
+
+        //update
+        Book updatedBook = bookDao.findAll().stream()
+                .filter(b -> Objects.equals(b.getTitle(), "Effective Java"))
+                .findFirst()
+                .get();
```

Comment:
This code may throw a NoSuchElementException if 'findFirst()' doesn't find any element. It's better to use 'ifPresent()' after 'findFirst()' to check if the book is present before calling 'get()'.